manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -2465,6 +2465,232 @@ spec:
                     local:
                       inline_string: "envoy.wasm.stats"
 ---
+# Source: istiod/templates/telemetryv2_1.16.yaml
```

Comment:
Do we remove the .13 files as they should be EOL by the time this branch is published as 1.16? Or do we wait until 1.13 is EOL in approx. 10 weeks?